doc/advanced-configuration/outgoing-connections.md

```diff
@@ -10,11 +10,12 @@ The interface for outgoing connections management was unified and is now availab
 * `elastic` - pool of connections to ElasticSearch server
 * `rdbms` - pool of connections to an RDBMS database
 * `rabbit` - pool of connections to a RabbitMQ server
-* `generic` - pool of generic workers not assosiated directly with a paritcualr connection (SNS, PushNotifications)
+* `ldap` - pool of connections to an LDAP server
+* `generic` - pool of generic workers not assosiated directly with a particular connection (SNS, PushNotifications)
```

Comment:
"assosiated" should be instead "associated"

Reply:
Again, not my change, good catch, will fix.